doc/getstarted.md

```diff
@@ -34,7 +29,6 @@ GitHub repo for directions on building.
   total size regression/improvement, and size regression/improvement by file and method.
 * jit-diff - Driver tool that implements a common dev flow.  Includes a configuration file
   for common defaults, and implements directory scheme for "installing" tools for use later.
-* jit-format - Run formatting tools on jit source code. Uses clang-tidy and clang-format.
```

Comment:
I've refactored it. The top-level README.md points to all the documentation for formatting. The getstarted.md is going to only cover diffs (and I'm planning to move it to the diffs page, or link to it from the diffs page).